Flowboard: new options and header formatting for small width tab

tmccormi opened the pull request:
Added options to hide more stuff and fixed header formatting to allow for more narrow tab presentation and use.

***BUT*** I think I broke something, looks pretty but it's not displaying any appts now :-)

@kchapple looking for someone to test.

tmccormi commented:
It's all working now.  Thanks Ken for the test pass.

@teryhill are you good with these enhancements?

tmccormi commented:
There is a fair amount of formatting clean up in here to make it a bit more readable.  Sorry, I know that makes diffs harder...

teryhill commented on interface/main/main_title.php:
```diff
@@ -117,7 +117,7 @@ function showhideMenu() {
 		<td align="right" class="text" style="vertical-align:text-bottom;"><a href='main_title.php' onclick="javascript:parent.left_nav.goHome();return false;" ><?php xl('Home','e'); ?></a>
 		&nbsp;|&nbsp;
-		<a href="http://open-emr.org/wiki/index.php/LibreEHR_4.2.1_Users_Guide" target="_blank" id="help_link" >
+		<a href="http://LibreHealth.io" target="_blank" id="help_link" >
```
You really should just remove this button. It is done in the about page.

tmccormi replied:
I'll just remove the file from this check in, it's unrelated anyway.

teryhill commented on interface/patient_tracker/patient_tracker.php:
```diff
@@ -158,9 +158,7 @@ function refreshbegin(first){
       refreshme();
     }
     setTimeout("refreshbegin('0')",parsetime);
-  <?php } else { ?>
```
What happens when there is no automatic refresh and you refresh it manually

tmccormi replied:
Didn't even know that was possible, what should happen?

tmccormi replied:
I just removed the return because a 'return' with a return value is meaningless php.  So it behaves the same way it did before.

tmccormi commented on interface/patient_tracker/patient_tracker.php:
```diff
@@ -86,7 +86,7 @@
 # go get the information and process it
 $appointments = fetch_Patient_Tracker_Events($from_date, $to_date, $provider, $facility, $form_apptstatus, $form_apptcat);
-$appointments = sortAppointments( $appointments, 'date','time' );
+$appointments = sortAppointments( $appointments,'date','time' );  // TODO - TIME IS NOT A PARAMETER for this function
```
The time sort is needed, but the function call only support 2 arguments, time is not one of them...

teryhill commented on interface/patient_tracker/patient_tracker.php:
```diff
@@ -206,9 +204,9 @@ function openNewTopWindow(newpid,newencounterid) {
 <span class="title"><?php echo xlt("Flow Board") ?></span>
 <body class="body_top" >
 <form method='post' name='theform' id='theform' action='<?php echo $action_page; ?>' onsubmit='return top.restoreSession()'>
-    <div id="flow_board_parameters">
```
Why ditch the CSS formating?

tmccormi replied:
Because it prevents auto wrapping the parameters when the window is too narrow, I tried several thing and nothing worked but leaving it off.    I'm not CSS expert.   We never did get the CSS to work well in LibreEHR.

teryhill commented on interface/patient_tracker/patient_tracker_status.php:
```diff
@@ -1,126 +1,129 @@
+<head>
+    <link rel="stylesheet" href="<?php echo $css_header; ?>" type="text/css">
+    <link rel="stylesheet" type="text/css" href="../../library/js/fancybox/jquery.fancybox-1.2.6.css" media="screen"/>
+    <script type="text/javascript" src="../../library/js/jquery.1.3.2.js"></script>
```
I would go ahead and change this to 1.9.1 and change it in the main tracker file

teryhill commented:
A couple of minor changes 
1) the popup to change the status with the room disabled looks a little squashed
2) add the new options in globals to the user preferences

tmccormi commented:
Good catch, thought I had added them to the user_prefs

Also.   The customer is asking for a visual in the flowboard for double
booking ...  Should I option that or do you think it would be OK?

Here's what it looks like..
[image: Inline image 1]

Tony McCormick, CTO
www.mi-squared.com
Support: 866-735-0897, Direct: 713-574-6709
My Calendar:  http://bit.ly/XznvDo

tóg go bog ē - Best advice from a pub owner in Ireland, to take it easy.



On Fri, Jan 20, 2017 at 5:33 PM, Terry Hill <notifications@github.com>
wrote:

> A couple of minor changes
>
>    1. the popup to change the status with the room disabled looks a
>    little squashed
>    2. add the new options in globals to the user preferences
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/LibreHealthIO/LibreEHR/pull/350#issuecomment-274222344>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AARci7NwtCWRN8yhDAruZSRLzM89Qpidks5rUWBMgaJpZM4Lnie1>
> .
>

-- 
Please be aware that e-mail communication can be intercepted in 
transmission or misdirected. Please consider communicating any sensitive 
information by telephone.  The information contained in this message may be 
privileged and confidential. If you are NOT the intended recipient, please 
notify the sender immediately with a copy to hipaa-security@mrsb-ltd.com and 
destroy this message.

teryhill commented:
The image did not make it. How are you going to tell double booking? Option is always best

tmccormi commented:
![doublebooked](https://cloud.githubusercontent.com/assets/285835/22171008/ab61f3c0-df3a-11e6-955c-70c0fea65ea6.png)

I compare the previously list date+time with the current one.

teryhill commented:
I have clients that double book with different statuses, Do you think that the statuses should be checked ?
I like the Idea. 
Do you think the date and time highlighted will get the attention better.

tmccormi commented:
Explain squished ... mine looks fine in Chrome/Linux

![statuschange](https://cloud.githubusercontent.com/assets/285835/22171086/b8846e74-df3b-11e6-80cd-9d9004a80ce5.png)

tmccormi commented:
Firefox is tighter ... wonder why (but not bad)

![firefox-statuschg](https://cloud.githubusercontent.com/assets/285835/22171120/217c83b2-df3c-11e6-88cd-202c201bf70c.png)

teryhill commented:
normal

![with room](https://cloud.githubusercontent.com/assets/4655366/22171135/9bd36766-df55-11e6-8b08-e5d523ae1966.PNG)


Squished

![squished](https://cloud.githubusercontent.com/assets/4655366/22171140/ac9f5ae6-df55-11e6-8211-7ab22bfaffb0.PNG)

tmccormi commented:
I think time is enough, but date and time would be OK too.

re statuses:  interesting, might complicate it more that I want to do right now though.  Would need some way to say which statuses are OK to DB which aren't.

Re squished .. you mean between the dialog and the buttons... go it.  Both are the same squished on my screen since I use a the light.css

tmccormi commented:
frankly this should be a fancybox not a full popup window anyway ... but not tonight.

tmccormi commented:
I'm curious why patient_tracker_status.php includes fancybox js but does not use it.

tmccormi commented:
I added DIV IDs, for future use to the sections, but removed CSS from the style sheets to preserve current behavior.   Added globals for double booking flag option.

tmccormi commented:
Gonna merge this now ...
